docs/contribute/code.md

```diff
@@ -136,6 +147,23 @@ If your contribution focuses on a bugfix and you want it featured in the next st
 - [Sync a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks/syncing-a-fork)
 - [Merge an upstream repository into your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks/merging-an-upstream-repository-into-your-fork)
 
+### Reproducing job failures
+
+If one of your CI jobs fails, it will print a message explaining how to reproduce the task locally. Typically that involves running the task against the right template:
+
+```shell
+yarn task --task e2e-tests --template=cra-default-ts --start-from=install
+```
+
+Typically it is a good idea to start from the `install` task to ensure your local code is completely up to date. If you reproduce the failure, you can try and make fixes, [compile them](#start-developing) with `build`, then rerun the task with `--start-from=auto`.
+ 
+ <div class="aside">
+
+<p>💡 The default instructions run the code "linked" which means built changes to Storybook code will be reflected in the sandbox right away (the next time you run the task). However CI runs in "unlinked" mode, which in rare cases will behave differently.</p>
```

Comment:
Could it be confusing for people to read "changes to Storybook code" ? Could they think like "does that mean changing the stories file of the example?" instead of "does that mean changing the source code of the Storybook libraries"

Reply:
```suggestion
<p>💡 The default instructions run the code "linked" which means built changes to Storybook library code will be reflected in the sandbox right away (the next time you run the task). However CI runs in "unlinked" mode, which in rare cases will behave differently.</p>
```